test/utilities/deployAndRetrieve.ts

```diff
@@ -62,9 +81,9 @@ export const validateCommand = async (
 export const calculateSpacer = (longestFullName: string, currentFileName: string): string => {
   let numberOfSpaces = 2;
   if (longestFullName.length < 'FULL_NAME'.length) {
-    numberOfSpaces += ('FULL_NAME'.length - currentFileName.length);
+    numberOfSpaces += 'FULL_NAME'.length - currentFileName.length;
   } else {
-    numberOfSpaces += (longestFullName.length - currentFileName.length);
+    numberOfSpaces += longestFullName.length - currentFileName.length;
   }
```

Comment:
so it would never actually be 2?  

gh won't let me do a suggestion on deleted line, but cosnt numberOfSpaces = 2 + longestFullName.length < 'FULL_NAME'.length ? ('FULL_NAME'.length - currentFileName.length) : (longestFullName.length - currentFileName.length)